### PR TITLE
Version cache unit test

### PR DESCRIPTION
**Pull request by mhegazy**
Port unit tests for versioncach.

### CODE REVIEW

**Comment by mhegazy** [comment]
@steveluc can you take a look.

File: tests/cases/unittests/versionCache.ts [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+                if (0 == (i % 4)) {
+                    let snap = svc.getSnapshot();
+                    let snapText = snap.getText(0, checkText.length);
+                    assert.equal(checkText, snapText);
+                }
+            }
+        });
+
+        it("Edit (average length 1/4th file size)", () => {
+            for (let i = 0; i < interationCount; i++) {
```
Comment (DanielRosenwasser) [inline tests/cases/unittests/versionCache.ts]: iterationCount

File: Jakefile [hunk 2]
```diff
@@ -143,7 +143,8 @@ var harnessSources = [
     "services/colorization.ts",
     "services/documentRegistry.ts",
     "services/preProcessFile.ts",
-    "services/patternMatcher.ts"
+    "services/patternMatcher.ts",
+	"versionCache.ts"
```
Comment (vladima) [inline Jakefile]: extra space\tab?

Comment (mhegazy) [inline Jakefile]: fixed

**Comment by steveluc** [comment]
:+1:   Thanks @mhegazy.

**Pull request merged**